README.md

```diff
@@ -879,33 +887,43 @@ def a(self, x=0):
 
 This project follows the
 [all-contributors](https://github.com/kentcdodds/all-contributors)
 specification. Contributions of any kind are welcome!
 
-1.  Check for open issues or open a fresh issue to start a discussion
-    around a feature idea or a bug. There is a 
-    [first-timers-only](https://github.com/gaphor/gaphas/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Afirst-timers-only)
-    tag for issues that should be ideal for people who are not very
-    familiar with the codebase yet.
-2.  Fork [the repository](https://github.com/gaphor/gaphas) on
-    GitHub to start making your changes to the **master** branch (or
-    branch off of it).
-3.  Write a test which shows that the bug was fixed or that the feature
-    works as expected.
-4.  Send a pull request and bug the maintainers until it gets merged and
-    published. :smile:
+1. Check for open issues or open a fresh issue to start a discussion
+   
```

Comment:
Why were the extra lines added in this section?

Reply:
Removed it.